src/main/scala/is/hail/methods/PCRelate.scala

```diff
@@ -170,6 +170,8 @@ class PCRelate(maf: Double, blockSize: Int) extends Serializable {
     gt != 0.0 && gt != 1.0 && gt != 2.0
 
   def apply(vds: VariantDataset, pcs: DenseMatrix): Result[M] = {
+    vds.requireUniqueSamples("pc_relate")
```

Comment:
pc-relate / IBD generate matrices with string keys, and I think that breaks (at least better safe than sorry)